packages/server/test/support/fixtures/projects/browser-extensions/devtools-ext/devtools.html

```diff
@@ -1,5 +1,14 @@
-<script type="text/javascript">
-chrome.devtools.inspectedWindow.onResourceAdded.addListener(() => {
-  chrome.devtools.inspectedWindow.eval('window.theExtensionLoaded = true')
-})
+<script>
+  function addProperty () {
+    chrome.devtools.inspectedWindow.onResourceAdded.addListener(() => {
+      chrome.devtools.inspectedWindow.eval('window.theExtensionLoaded = true')
+    })
+  }
+
+  // sometimes chrome.devtools is undefined, most likely bug in chromium :|
```

Comment:
the `if` checks for `inspectedWindow`, so i'm guessing you meant this?

```suggestion
  // sometimes chrome.devtools.inspectedWindow is undefined, most likely bug in chromium :|
```

is there a bug report that you can link to here? or is it just a race condition that the inspected page hasn't loaded yet and so `inspectedWindow` is undefined?

Reply:
whoops

Reply:
no bug report, just noticed it failing in CI: https://circleci.com/gh/cypress-io/cypress/314207?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link